src/plot.js

```diff
@@ -239,6 +239,7 @@ export function plot(options = {}) {
   // Compute value objects, applying scales and projection as needed.
   for (const [mark, state] of stateByMark) {
     state.values = mark.scale(state.channels, scales, context);
+    state.values.data = state.data; // expose transformed data for advanced usage
```

Comment:
It will be immensely useful in the render API to always have access to mark’s (arrayified, possibly transformed) data. I just wonder why it appears here, as it seems unrelated to this PR. Needs to be added to the documentation #1811.

Reply:
This was needed in an earlier variation of this PR where you could specify the format as a function of data. I don’t think it’s needed anymore, so I can remove it and we can add it as a separate PR when we are ready.